.travis.yml

```diff
@@ -33,25 +33,32 @@ matrix:
     - php: 7.3
       env:
         - COMPOSER_FLAGS="--prefer-stable --prefer-lowest" DEPENDENCIES="symfony/validator:^3.4.4 symfony/templating:^3.4.4"
```

Comment:
please remove the lowest version build with php 7.3. we only want to do that with the oldest and the newest php version.